README.md

```diff
@@ -88,22 +98,50 @@ export interface ErrorMessages {
   callParenthesesMissing: string;
   /** This variable is not defined. */
   undefinedVariable: string;
-  /** Expected at least one parameter in a function call. */
-  expectedAtLeastOneParameter: string;
-  /** Expected that many parameters, found this many. */
-  parameterCountMismatch: (thatMany: number, thisMany: number) => string;
+  /** Expected exactly that many parameters, found this many. */
+  expectedExactArgumentCount: (count: number) => string;
+  /** Expected at least that many parameters, found this many. */
+  expectedMinimumArgumentCount: (count: number) => string;
```

Comment:
issue: these two errors have `count` as a parameter, and it's not clear whether it refers to `that many` or `this many`. For example, for `expectedExactArgumentCount` I would like to supply a string `"Expected 3 arguments, found 2"`, but for that I need both 3 and 2 as parameters

issue: since we have minimum and range, I think it'd be reasonable to also add maximum

suggestion: add another parameter to these functions and add a new error message `expectedMaximumArgumentCount`